src/main.rs

```diff
@@ -409,9 +427,13 @@ fn extract_time_constraints(opts: &Opts) -> Result<Vec<TimeFilter>> {
 
 fn ensure_use_hidden_option_for_leading_dot_pattern(
     config: &Config,
-    pattern_regex: &str,
+    pattern_regexps: &[String],
 ) -> Result<()> {
-    if cfg!(unix) && config.ignore_hidden && pattern_matches_strings_with_leading_dot(pattern_regex)
+    if cfg!(unix)
+        && config.ignore_hidden
+        && pattern_regexps
+            .iter()
+            .any(|pat| pattern_matches_strings_with_leading_dot(pat))
```

Comment:
does this work?

```suggestion
            .any(pattern_matches_strings_with_leading_dot)
```

Reply:
I'm getting: `rustc [E0631]: expected signature of fn(&std::string::String) -> _`

Reply:
ok, then let's leave this.